src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -1747,16 +1747,7 @@ public function testOrderWithGhosts()
 
     public function testOrderNonExistingSource()
     {
-        $data = [
-            [
-                'title' => 'test1',
-                'url' => '/test1',
-            ],
-        ];
-
         $client = $this->createAuthenticatedClient();
-        $client->request('POST', '/api/nodes?webspace=sulu_io&language=en', $data[0]);
```

Comment:
This request was erroneous already before, but the type of error changed. The error it threw before was somehow caught by symfony internals, but it also didn't work, because the `template` parameter was missing.

However, I have seen that to cover this test the first request was not necessary at all.